kernel/qGraph.ml

```diff
@@ -0,0 +1,170 @@
+(************************************************************************)
+(*         *      The Rocq Prover / The Rocq Development Team           *)
+(*  v      *         Copyright INRIA, CNRS and contributors             *)
+(* <O___,, * (see version control and CREDITS file for authors & dates) *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
+open Sorts
+
+module ElimTable = struct
+  open Quality
+
+  let const_eliminates_to q q' =
+    match q, q' with
+    | QType, _ -> true
+    | QProp, (QProp | QSProp) -> true
+    | QSProp, QSProp -> true
+    | _ -> false
```

Comment:
```suggestion
    | (QType | QProp | QSProp), _ -> false
```
that way ocaml will tell us this needs to be changed if we add another constant quality